models/user_openid_test.go

```diff
@@ -15,18 +15,20 @@ func TestGetUserOpenIDs(t *testing.T) {
 
 	oids, err := GetUserOpenIDs(int64(1))
 	if assert.NoError(t, err) {
-		assert.Len(t, oids, 2)
-		assert.Equal(t, oids[0].URI, "https://user1.domain1.tld/")
-		assert.False(t, oids[0].Show)
-		assert.Equal(t, oids[1].URI, "http://user1.domain2.tld/")
-		assert.True(t, oids[1].Show)
+		if assert.Len(t, oids, 2) {
```

Comment:
Maybe it'd be better to combine the two `if` statement to reduce nesting
```
if assert.NoError(t, err) && assert.Len(t, oids, 2) {
...
}
```

Reply:
You're right